src/main/java/uk/gov/hmcts/reform/pip/rules/errorhandling/GlobalExceptionHandler.java

```diff
@@ -32,4 +34,38 @@
         return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponse);
     }
 
+    /**
+     * Template exception handler, that handles a custom CourtNotFoundException,
+     * and returns a 404 in the standard format.
+     * @param ex The exception that has been thrown.
+     * @return The error response, modelled using the ExceptionResponse object.
+     */
+    @ExceptionHandler(CourtNotFoundException.class)
+    public ResponseEntity<ExceptionResponse> handleCourtNotFound(
+        CourtNotFoundException ex) {
+
+        ExceptionResponse exceptionResponse = new ExceptionResponse();
+        exceptionResponse.setMessage(ex.getMessage());
+        exceptionResponse.setTimestamp(LocalDateTime.now());
+
+        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponse);
+    }
+
+    /**
+     * Template exception handler, that handles a custom CourtNotFoundException,
+     * and returns a 404 in the standard format.
+     * @param ex The exception that has been thrown.
+     * @return The error response, modelled using the ExceptionResponse object.
+     */
+    @ExceptionHandler(ParseException.class)
+    public ResponseEntity<ExceptionResponse> handleParseException(
+        ParseException ex) {
+
+        ExceptionResponse exceptionResponse = new ExceptionResponse();
+        exceptionResponse.setMessage(ex.getMessage());
+        exceptionResponse.setTimestamp(LocalDateTime.now());
+
+        return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(exceptionResponse);
```

Comment:
would this be helpful to someone seeing this?

Reply:
done